census_blueprints/execute_sync.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_SYNC_REFRESH_ERROR = 210
+EXIT_CODE_UNKNOWN_ERROR = 211
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--sync-id', dest='sync_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def execute_sync(sync_id, access_token):
+    """
+    Executes/starts a Census Sync
+    """
+    sync_post_api = f"https://bearer:{access_token}@app.getcensus.com/api/v1/syncs/{sync_id}/trigger"
+    api_headers = {
+        'Content-Type' : 'application/json'
+        }
+    try:
```

Comment:
Let's define the variable `sync_trigger_json = None` before the `try`. I guess Python allows it, but it feels wrong to have a variable fist declared inside the inner try block and then be referenced outside of it.